.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug report
+about: Report a bug
+
+---
+<!-- 
+Please search GitHub for a similar issue or PR before submitting.
```

Comment:
Looking good, let's clarify a bit more by saying something like:

_Please, before creating an issue or submitting a PR, check that your issue is not already fixed in the latest stable version and that a similar issue or PR is not reported already (check also closed issues)._